packages/python/algokit_utils/tests/test_utils.py

```diff
@@ -0,0 +1,80 @@
+from typing import override
+import typing
+from algokit_utils.algokit_http_client import HttpClient, HttpMethod, HttpResponse
+from algokit_utils.algokit_transact_ffi import SignedTransaction, Transaction
+from algokit_utils import AlgodClient, TransactionSigner
+from algokit_utils.algokit_utils_ffi import (
+    CommonParams,
+    Composer,
+    PaymentParams,
+    TransactionSignerGetter,
+)
+from algosdk.mnemonic import to_private_key
+from nacl.signing import SigningKey
+import base64
+import pytest
+
+MN = "limb estate enhance elegant merry worry spell trophy elegant lab truly step enter destroy split leave beach chalk slight they ignore square tower abandon rough"
+SEED_B64: str = to_private_key(MN)  # type: ignore
+SEED_BYTES = base64.b64decode(SEED_B64)
+KEY = SigningKey(SEED_BYTES[:32])
```

Comment:
The python SDK is poorly documented for how the to_private_key method actually works, but I think this should work